### ISSUE TITLE
Add ability to test scraper configuration

### ISSUE THREAD

**Issue by dtslvr** [ghostfolio/ghostfolio#2292]
### Outline

1. Add a button to the [asset profile dialog](https://github.com/ghostfolio/ghostfolio/tree/main/apps/client/src/app/components/admin-market-data/asset-profile-dialog) to test a scraper configuration
2. Make a request to `/api/v1/gather/MANUAL/:symbol/:dateString` (where `dateString` is today)
3. Show the result (`marketPrice`) as an alert

### Mockup

<img width="795" alt="scraper-configuration" src="https://github.com/ghostfolio/ghostfolio/assets/4159106/0ad7e9f3-483b-4147-8b16-356c01ccefe3">

**Comment by Siddharth9890** [comment]
i can work on this @dtslvr please?

**Comment by dtslvr** [comment]
Any progress to share on this @Siddharth9890?

**Comment by Siddharth9890** [comment]
Sorry @dtslvr currently i have exams so i am busy with it you can assign it to someone else if needed. Thank you !

**Comment by M27afk** [comment]
Can I get assigned on this? @dtslvr

**Comment by dtslvr** [comment]
> 1. The Scraper configuration is available only for Manual assets right?
> 2. Is @Post('gather/:dataSource/:symbol/:dateString') in admin.controller.ts the right route to be called?
> 3. How is the data in the scraper configuration input related to the API call?

1. Yes, scraper configuration is only used where `dataSource` is `MANUAL`
2. Yes
3. Behind the scenes, it calls `getHistorical()` with the relevant data provider ([ManualService](https://github.com/ghostfolio/ghostfolio/blob/main/apps/api/src/services/data-provider/manual/manual.service.ts#L65) in this case) and uses the scraper configuration

**Comment by M27afk** [comment]
Can I get sample Scraper Configuration of a Sample manual asset and expected output for testing purpose?

**Comment by dtslvr** [comment]
> Can I get sample Scraper Configuration of a Sample manual asset and expected output for testing purpose?

Sure! This `scraperConfiguration` should return the most recent price:

```
{
  "url": "https://zkb-finance.mdgms.com/products/funds/zkb/detail.html?LANG=en&FI_ID_NOTATION=287639517",
  "selector": ".fi_table_module tr:nth-of-type(1) td:nth-of-type(3)"
}
```

<img width="694" alt="scraper-configuration-zkb" src="https://github.com/ghostfolio/ghostfolio/assets/4159106/b38e1bac-c996-4a6b-b269-81fa871f341b">

**Comment by dtslvr** [comment]
Implemented via #2808.
